docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+[//]: # (title: Troubleshooting)
+
+Most of the browsers support WebAssembly. For recent browser versions, WebAssembly works by default. However,
+for older browser versions, you still need to update the browser settings.
+
+To run a Kotlin/Wasm project, check if you need to update the settings of the target environment:
+
+<tabs>
```

Comment:
I'm wondering if it will look better without tabs 🤔

Reply:
As we're presenting more than 2 browsers, I would say it's more user-friendly to keep them in tabs. Users can quickly switch between different sections without scrolling excessively, which helps in terms of navigation and segmentation of information.

Reply:
It seems to me that tab-less could be more discoverable.

Other concerns:
* Content inside an inactive tab is not searchable via builtin find on the page.
* There is a relative link to tabs, but I'm not sure if they are stable.

Reply:
[Minor]

Reply:
Ok, I tried 3 styles (tabs, collapsed paragraphs, and expanded information). In the end, expanded information was easy to navigate because this is a stand-alone page and there is no more information or subsections below. In case we add more sections and subsections below later, we can think of a solution.

Reply:
Thank you for experimenting!